Package.swift

```diff
@@ -8,14 +8,17 @@ let package = Package(
     .executable(name: "pointfreeco", targets: ["pointfreeco"]),
   ],
   dependencies: [
-    .package(url: "https://github.com/IBM-Swift/Kitura.git", from: "2.1.0"),
+    .package(url: "https://github.com/IBM-Swift/Kitura.git", from: "2.0.0"),
+    .package(url: "https://github.com/IBM-Swift/Kitura-Compression", from: "2.0.1"),
```

Comment:
`Kitura-Compression` depends on `Kitura` 2.0 instead of 2.1, so had to "downgrade" kitura :/